sale_order_line_input/views/sale_order_line_view.xml

```diff
@@ -0,0 +1,264 @@
+                        'pricelist': pricelist_id,
+                        'uom':product_uom,
+                        'company_id': company_id,
+                        'default_lst_price': price_unit,
+                        'default_description_sale': name
+                    }"
+                    options="{
+                        'no_open': True,
+                    }"
+                    domain="[('sale_ok', '=', True), '|', ('company_id', '=', False), ('company_id', '=', company_id)]"
+                    widget="sol_product_many2one"
+                />
+                <field
+                    name="product_template_id"
+                    string="Product"
+                    column_invisible="1"
+                    readonly="not product_updatable"
+                    required="not display_type"
+                    context="{
+                        'partner_id': order_partner_id,
+                        'quantity': product_uom_qty,
+                        'pricelist': pricelist_id,
+                        'uom':product_uom,
+                        'company_id': company_id,
+                        'default_list_price': price_unit,
+                        'default_description_sale': name
+                    }"
+                    options="{
+                        'no_open': True,
+                    }"
+                    domain="[('sale_ok', '=', True), '|', ('company_id', '=', False), ('company_id', '=', company_id)]"
+                    widget="sol_product_many2one"
+                />
+                <field name="name" widget="section_and_note_text" optional="show" />
+                <field
+                    name="analytic_distribution"
+                    widget="analytic_distribution"
+                    optional="hide"
+                    groups="analytic.group_analytic_accounting"
+                    options="{'product_field': 'product_id', 'business_domain': 'sale_order'}"
+                />
+                <field
+                    name="product_uom_qty"
+                    decoration-info="(not display_type and invoice_status == 'to invoice')"
+                    decoration-bf="(not display_type and invoice_status == 'to invoice')"
+                    context="{
+                        'partner_id': order_partner_id,
+                        'quantity': product_uom_qty,
+                        'pricelist': pricelist_id,
+                        'uom': product_uom,
+                        'company_id': company_id
+                    }"
+                />
+                <field
+                    name="qty_delivered"
+                    decoration-info="(not display_type and invoice_status == 'to invoice')"
+                    decoration-bf="(not display_type and invoice_status == 'to invoice')"
+                    string="Delivered"
+                    invisible="state != 'sale'"
+                    readonly="'qty_delivered_method' != 'manual'"
```

Comment:
I think it must be evaluated as an expression instead of a string.
```suggestion
                    readonly="qty_delivered_method != 'manual'"
```